features/2025-08-14-terraform-settings.md

```diff
@@ -0,0 +1,554 @@
+| credential_helper                                   | External program for authenticating                          | ❌                   |
+| disable_checkpoint and disable_checkpoint_signature | Disables checking with Hashicorp about upgrade and security updates | ❌                   |
+| plugin_cache_dir                                    | Location of plugin cache                                     | ❌                   |
+| provider_installation                               | Specifies the location of providers                          | ✅                   |
+
+The entire set of properties for credentials and provider_installation are supported. See the [Terraform documentation](https://developer.hashicorp.com/terraform/cli/config) for the full set of properties.
+
+Unsupported settings are not part of the TerraformSettings resource schema, so the TerraformSettings resource will not deploy if unsupported settings are specified. 
+
+### User Story 5 – Configure Terraform Backend
+
+***As a platform engineer, I need to configure the Terraform backend state store.***
+
+**Summary**
+
+The Terraform backend is the state store for Terraform. Today, Radius uses an opinionated Kubernetes backend. Platform engineers need the ability to specify an existing backend. Terraform backends are specified in the `terraform` block similar to how providers are specified. 
+
+Radius will support all backend types supported by Terraform except the local backend and remote types. Local does not make sense when running Terraform via Radius and remote has been replaced with Terraform Cloud.
+
+There will be two tiers of support:
+
+* **Tier 1** – Properties available in the TerraformSettings resource, integrated authentication, tested by the Radius project
+* **Tier 2** – Only properties available in the TerraformSettings resource. It is left to future development to implement authentication for these backend types.
+
+The backend types include:
+
+| Tier 1                                 | Tier 2                                             |
+| -------------------------------------- | -------------------------------------------------- |
+| Kubernetes (the default)               | Alibaba Cloud Object Storage Service (`oss`)       |
+| AWS S3 (`s3`)                          | Consul (`consul`)                                  |
+| Azure Blob Storage Account (`azurerm`) | Google Cloud Storage (`gcs`)                       |
+|                                        | HTTP (`http`)                                      |
+|                                        | Oracle Cloud Infrastructure Object Storage (`oci`) |
+|                                        | PostgreSQL database (`pg`)                         |
+|                                        | Tencent Cloud Object Storage (`cos`)               |
+
+As with the Terraform CLI settings, Terraform backend is configured via the TerraformSettings resource.
```

Comment:
So to apply the Terraform backend config, one would just run `rad deploy terraformS3Backend.bicep`?